wallettool/src/main/java/org/bitcoinj/wallettool/WalletTool.java

```diff
@@ -1098,6 +1045,66 @@ private void dumpWallet() throws BlockStoreException {
         }
     }
 
+    private static int rawDumpWallet(File walletFile) throws IOException {
+        try (FileInputStream stream = new FileInputStream(walletFile)) {
+            Protos.Wallet proto = WalletProtobufSerializer.parseToProto(stream);
+            proto = attemptHexConversion(proto);
+            System.out.println(proto.toString());
+            return 0;
+        }
+    }
+
+    private int send(){
+        if (feePerVkbStr != null && feeSatPerVbyteStr != null) {
+            System.err.println("--fee-per-kb and --fee-sat-per-byte cannot be used together.");
+            return 1;
+        } else if (outputsStr != null) {
+            Coin feePerVkb;
+            if (feePerVkbStr != null)
+                feePerVkb = parseCoin(feePerVkbStr);
+            else if (feeSatPerVbyteStr != null)
+                feePerVkb = Coin.valueOf(Long.parseLong(feeSatPerVbyteStr) * 1000);
+            else
+                feePerVkb = null;
+            if (selectAddrStr != null && selectOutputStr != null) {
+                System.err.println("--select-addr and --select-output cannot be used together.");
+                return 1;
+            }
+            CoinSelector coinSelector;
+            if (selectAddrStr != null) {
+                Address selectAddr;
+                try {
+                    selectAddr = wallet.parseAddress(selectAddrStr);
+                } catch (AddressFormatException x) {
+                    System.err.println("Could not parse given address, or wrong network: " + selectAddrStr);
+                    return 1;
+                }
+                final Address validSelectAddr = selectAddr;
+                coinSelector = CoinSelector.fromPredicate(candidate -> {
+                    try {
+                        return candidate.getScriptPubKey().getToAddress(net).equals(validSelectAddr);
+                    } catch (ScriptException x) {
+                        return false;
+                    }
+                });
+            } else if (selectOutputStr != null) {
+                String[] parts = selectOutputStr.split(":", 2);
+                Sha256Hash selectTransactionHash = Sha256Hash.wrap(parts[0]);
+                int selectIndex = Integer.parseInt(parts[1]);
+                coinSelector = CoinSelector.fromPredicate(candidate ->
+                        candidate.getIndex() == selectIndex && candidate.getParentTransactionHash().equals(selectTransactionHash)
```

Comment:
The indentation is wrong here. It should be 4 spaces. (The original was indented by 5 which is also wrong -- so that's probably how why it was changed by your IDE to 8)